ISISNeutronMuon/MDANSE
# Make headers GPL compliant and removed __pkginfo__

**Description of work**
The MDANSE and MDANSE_GUI codes are both licensed under GPLv3. This requires us to use the recommended license notice throughout the code, as per https://www.gnu.org/licenses/gpl-howto.html. Also removed __pkginfo__ files.

**Fixes**
The headers have been replaced with the GPL-compliant headers. The contents of the copyright notice and authors list are not affected by the change.

**To test**
There should be no change to the code operation. You are welcome to check if there are still some files that need updating.
